solutions/swb-reference/integration-tests/tests/multiStep/environment.test.ts

```diff
@@ -29,6 +38,207 @@ describe('multiStep environment test', () => {
       PROJ: expect.anything() // PROJ should be defined
     });
 
-    // TODO: Add connect, stop, get, terminate test
+    //Create Environment B
+    console.log('Creating Environment B');
+    const { data: environmentB } = await adminSession.resources.environments.create();
+    expect(environmentB).toMatchObject({
+      id: expect.stringMatching(uuidRegExp),
+      instanceId: '', // empty string because instanceId value has not been propagated by statusHandler yet
+      provisionedProductId: '', // empty string because provisionedProductId  has not been propagated by statusHandler yet
+      status: 'PENDING',
+      ETC: expect.anything(), //ETC should be defined
+      PROJ: expect.anything() // PROJ should be defined
+    });
+
+    //Verify Environment A was started and is available
+    console.log('Verify Environment A was started and is available');
+    await poll(
+      async () => await adminSession.resources.environments.environment(environmentA.id).get(),
```

Comment:
Can this line be 
`async () => return adminSession.resources.environments.environment(environmentA.id).get()`

I think if you're returning a promise, you don't need to add `await`.  Can you confirm?

Reply:
thats correct, I removed it and passed